libkineto/src/CuptiActivityProfiler.cpp

```diff
@@ -944,6 +948,9 @@ void CuptiActivityProfiler::stopTraceInternal(
 
 void CuptiActivityProfiler::resetInternal() {
   resetTraceData();
+  for (auto& session : sessions_) {
+    session->reset();
```

Comment:
If you see resetTraceData() in the line above the sessions_ object is cleared ! So after line 951 sessions_ is empty.
This is why i was asking if you really need a reset() API ?

Reply:
Thanks for your correction. I didn't noticed that sessions are cleared in resetTraceData() before. Already remove this addition.